src/generic/grid.cpp

```diff
@@ -4361,16 +4415,72 @@ void wxGrid::DoStartResizeRowOrCol(int col, int size)
 
 void wxGrid::ProcessCornerLabelMouseEvent( wxMouseEvent& event )
 {
-    if ( event.LeftDown() )
+
+    if ( CheckIfDragCancelled(&event) )
+        return;
+
+    // if label resize is allowed:
+    //  - check the mouse position is at right or bottom edge
+    //  - initialize oper to correct operation type
+    bool atBottomEdge = false, atRightEdge = false, atLabelEdge = false;
+    const wxPoint posEvent = event.GetPosition();
+    int edge = FromDIP(WXGRID_LABEL_EDGE_ZONE);
+    const wxGridOperations* oper = nullptr;
```

Comment:
This is a memory leak AFAICS because `oper` is never deleted. Please use

```suggestion
    std::unique_ptr<wxGridOperations> oper;
```

instead.